layouts/index.html

```diff
@@ -2,7 +2,10 @@
 <div class="container mx-auto">
   {{ partial "homepage/role_slider.html" . }}
   {{ partial "homepage/recents.html" . }}
```

Comment:
While we're going through this PR, I might ask you to hold off on some of these partials – get that the home page looks empty without some of these, but at the same time we've also got to focus on getting the theme good enough to move over to the primary domain/repo.

Reply:
Sounds good. I believe I'll comment out the "Latest Guides" and the "Community Spotlight" sections as that is what the consensus is from the dev channel :)